### PR TITLE
Change atmospheres to render using batch compatible pipeline and shaders

### PR DESCRIPTION
**Pull request by ZappoMan**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 268bb370ccafc9d395cb40e600005410e8bb092c available at: https://gist.github.com/fbb9d1a5a0e73597626c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: af5a910f5819c07ce01baf942553f9a7885c6f29 available at: https://gist.github.com/fbb9d1a5a0e73597626c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 467609f2b69e0b6c9382071a99523f978735b5b2 available at: https://gist.github.com/fbb9d1a5a0e73597626c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.dmg

File: libraries/render-utils/src/SkyFromAtmosphere.slf [hunk 1]
```diff
@@ -102,7 +104,9 @@ void main (void)
     
     float fCos = dot(v3LightPos, v3Direction) / length(v3Direction);
     float fMiePhase = 1.5 * ((1.0 - g2) / (2.0 + g2)) * (1.0 + fCos*fCos) / pow(1.0 + g2 - 2.0*g*fCos, 1.5);
-    gl_FragColor.rgb = frontColor.rgb + fMiePhase * secondaryFrontColor.rgb;
-    gl_FragColor.a = gl_FragColor.b;
-    gl_FragColor.rgb = pow(gl_FragColor.rgb, vec3(1.0/2.2));
+    
+    vec3 finalColor = frontColor.rgb + fMiePhase * secondaryFrontColor.rgb;
+    gl_FragData[0].rgb = finalColor;
```
Comment (Atlante45) [inline libraries/render-utils/src/SkyFromAtmosphere.slf]: This line is not required anymore.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 689e10cc27185c0d17973e28fdbe01eba06e3f75 available at: https://gist.github.com/fbb9d1a5a0e73597626c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: e9bf553254c2edf9e80caf691c94a5e79db554a0 available at: https://gist.github.com/fbb9d1a5a0e73597626c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fb59939cf4b2781e045b07837fda4a1f06c1db30 available at: https://gist.github.com/fbb9d1a5a0e73597626c<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-5143.dmg

**Pull request merged**